app/views/layouts/application.html.erb

```diff
@@ -46,6 +46,7 @@
             Apply for postgraduate teacher training
           </a>
           <p><%= current_candidate.email_address unless current_candidate.nil? %></p>
+          <a href=<%= candidate_interface_start_path %>><%= 'Sign out' unless current_candidate.nil? %></a>
```

Comment:
This could be a `<%= link_to candidate_interface_start_path, "Sign out", class: "govuk-link" if current_candidate.present? %>` instead.

There's some changes to the line above in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/215 so this is likely to conflict, sry 😬

Reply:
Ah yes, we definitely can refactor! We're in the process of TDDing so I believe in the end this will be the Devise sign out path instead.

No worries! We can get your PR merged in this morning and fix possible conflicts after. 👍

Reply:
You can even use the `link_to_if` method:

```diff
- <%= link_to candidate_interface_start_path, "Sign out", class: "govuk-link" if current_candidate.present? %>
+ <%= link_to_if current_candidate, candidate_interface_start_path, "Sign out", class: "govuk-link" %>
```

Reply:
I didn't know that was a thing, that's cool! 🚀

Reply:
Sure, I thought as well because the PR is in draft that it will be refactored anyway 😄